modules/06 Angular/Assignments/15/answer15/src/app/box/box.component.html

```diff
@@ -0,0 +1,11 @@
+<div class="box" [ngClass]="{'clicked': clicked}"
+[style.width.px]="likeAmount > 10 ? '220' : '200'" 
```

Comment:
Get rid of login in the templates
https://angular.io/guide/styleguide#style-05-17